runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -96,6 +101,11 @@ public void onGameStateChanged(GameStateChanged event)
 			storeOriginalDots();
 			replaceMapDots();
 		}
+
+		if (event.getGameState() == GameState.LOGIN_SCREEN)
```

Comment:
Adding a minimap zoom changed event to hook into would be a better way to go about this. (would require an internal change though)